### PR TITLE
Add 99DOTS unenroll patient repeater

### PR DESCRIPTION
**Pull request by proteusvacuum**
https://trello.com/c/rDA0sCps/283-unenroll-patients-from-99dots-in-enikshay-is-reflected-in-99dots
https://docs.google.com/document/d/14D_kR6Z4TzSP5SPEDY-DTeViA_NQW7ipQgEtT8M8qc0/edit#heading=h.9oqjw84rfhsw

@mkangia 
@nickpell 
@sravfeyn

### CODE REVIEW

**Comment by gdeflaux** [comment]
Is it possible for someone else than Farid to fix the Travis build error and merge the PR?

@mkangia
@nickpell
@sravfeyn

**Comment by mkangia** [comment]
fix one test case i could and merged master for others because there was something causing all PR's to fail which was reverted later and possibly caused failures here as well.

File: custom/enikshay/integrations/ninetyninedots/tests/test_payload_generators.py [hunk 1]
```diff
@@ -327,6 +327,7 @@ def test_get_payload_closed_case(self):
         self.assertEqual(self._get_actual_payload(cases), expected_payload)
 
     def test_get_payload_source_changed(self):
+        self.episode.attrs['update']['dots_99_enabled'] = 'false'
```
Comment (proteusvacuum) [inline custom/enikshay/integrations/ninetyninedots/tests/test_payload_generators.py]: oops. thanks for fixing!

**Comment by proteusvacuum** [comment]
There's not much point merging this one before the new year as the 99DOTS endpoint isn't live yet, so we'll just get 404's for every request.

**Comment by proteusvacuum** [comment]
@mkangia could you give this a 👍 if you think this is g2g?

File: custom/enikshay/integrations/ninetyninedots/repeater_generators.py [hunk 2]
```diff
@@ -218,3 +218,52 @@ def handle_failure(self, response, episode_case, repeat_record):
...
+class UnenrollPatientPayloadGenerator(NinetyNineDotsBasePayloadGenerator):
+    deprecated_format_names = ('case_json',)
+
+    def get_payload(self, repeat_record, episode_case):
+        domain = episode_case.domain
+        person_case = get_person_case_from_occurrence(
+            domain, get_occurrence_case_from_episode(
+                domain, episode_case.case_id
+            ).case_id
+        )
```
Comment (mkangia) [inline custom/enikshay/integrations/ninetyninedots/repeater_generators.py]: Any reason for not using `get_person_case_from_episode` instead?

File: custom/enikshay/integrations/ninetyninedots/repeater_generators.py [hunk 3]
```diff
@@ -218,3 +218,52 @@ def handle_failure(self, response, episode_case, repeat_record):
...
+        }
+        return json.dumps(payload)
+
+    def handle_success(self, response, episode_case, repeat_record):
+        if response.status_code == 200:
+            update_case(
+                episode_case.domain,
+                episode_case.case_id,
+                {
+                    "dots_99_registered": "false",
```
Comment (mkangia) [inline custom/enikshay/integrations/ninetyninedots/repeater_generators.py]: Do we want to differentiate patients that
1. failed dots 99 notification.
2. unenrolled notification

**Review by mkangia — APPROVED** [review]
@proteusvacuum sorry for the late review on this, just left a couple of comments but no blocker

**Pull request merged**